config.yaml

```diff
@@ -1,136 +1,184 @@
 teams:
-- name: architecture-sig-maintainers
-  maintainers:
-  - tkuhrt
-- name: bifold-wallet-maintainers
-  maintainers:
-  - jleach
-  members:
-  - amanji
-  - bryce-mcmath
-  - cvarjao
-  - jcdrouin21
-  - MosCD3
-  - thiagoromanos
-  - wadeking98
-- name: credential-comparison-format-sig-committers
-  maintainers:
-  - cre8
-  - tkuhrt
-  - tlodderstedt
-- name: credential-comparison-format-sig-admins
-  maintainers:
-  - cre8
-- name: credo-admins
-  maintainers:
-  - TimoGlastra
-- name: credo-maintainers
-  maintainers:
-  - jakubkoci
-  - TimoGlastra
-  members:
-  - berendsliedrecht
-  - genaris
-  - karimStekelenburg
-- name: digital-wallet-and-agent-overviews-sig-maintainers
-  maintainers:
-  - cre8
-  members:
-  - samuelmr
-- name: digital-wallet-and-agent-overviews-sig-admins
-  maintainers:
-  - cre8
-- name: lf-employees
-  maintainers:
-  - ryjones
-  - strawberry-baked-alaska
-  - tbenzies
-  members:
-  - SeanBohan
-  - mkdolan
-- name: owl-mobile-wallet-test-harness-admins
-  maintainers:
-  - nodlesh
-- name: owl-mobile-wallet-test-harness-maintainers
-  maintainers:
-  - nodlesh
-- name: safe-wallet-sig-maintainers
-  maintainers:
-  - tkuhrt
-  members:
-  - skounis
-  - darrellodonnell
-- name: tac
-  maintainers:
-  - tkuhrt
-  members:
-  - davidz25
-  - rolsonquadras
-  - petecool
-  - aceshim
-  - skounis
-  - swcurran
-  - wenjing
+  - name: acapy-admins
+    maintainers:
+      - tkuhrt
+    members:
+      - WadeBarnes
+      - dhh1128
+      - TimoGlastra
+      - andrewwhitehead
+      - dbluhm
+      - swcurran
+  - name: acapy-committers
+    maintainers:
+      - tkuhrt
+    members:
+      - TelegramSam
+      - andrewwhitehead
+      - swcurran
+      - TimoGlastra
+      - WadeBarnes
+      - amanji
+      - chumbert
+      - dbluhm
+      - esune
+      - ianco
+      - jamshale
+      - ff137
+      - thiagoromanos
+      - PatStLouis
+  - name: acapy-contributors
+    maintainers:
+      - tkuhrt
+    members:
+      - TelegramSam
+      - WadeBarnes
+      - dhh1128
+      - nage
+      - swcurran
+      - Jsyro
+      - aholovko
+      - auer-martin
+      - baha-ai
+      - dbluhm
+      - esune
+      - ianco
+      - jamshale
+      - rolsonquadras
+      - troyronda
+      - wadeking98
+  - name: architecture-sig-maintainers
+    maintainers:
+      - tkuhrt
+  - name: bifold-wallet-maintainers
+    maintainers:
+      - jleach
+    members:
+      - amanji
+      - bryce-mcmath
+      - cvarjao
+      - jcdrouin21
+      - MosCD3
+      - thiagoromanos
+      - wadeking98
```

Comment:
Wade King (`wadeking98`) is no longer with the BC Wallet team.

Reply:
done